README.md

```diff
@@ -73,6 +73,8 @@ export class AppHomeComponent {
 
 - `autoUpload` Auto start upload when files added. Default value: `true`
 
+- `storeIncompleteUploadUrl` Keep an incomplete upload URL for allow resuming after browser restart. Default value: `true`
```

Comment:
Should this be added also to the `UploaderOptions` interface?

Reply:
https://github.com/kukhariev/ngx-uploadx/blob/7be8624f82900d1d6babfb94073c3aba4bf79154/src/uploadx/lib/interfaces.ts#L127-L132